src/library/itunes/itunesxmlimporter.cpp

```diff
@@ -65,6 +67,9 @@ ITunesImport ITunesXMLImporter::importLibrary() {
     ITunesImport iTunesImport;
     bool isMusicFolderLocatedAfterTracks = false;
 
+    // In sandboxed builds we have to obtain an access token
+    auto access = mixxx::FileAccess(mixxx::FileInfo(m_xmlFilePath));
```

Comment:
Two notes on this:

- It would be nice if someone could verify that this still works properly on Windows/Linux (though I'd be very surprised if there were any issues given that the browse feature obtains access tokens in the same manner).
- We should probably also do this for the Serato/Traktor/... integrations